### PR TITLE
blockstore: change unit of bloom filter to byte from bits

### PR DESCRIPTION
**Pull request by Kubuxu**
License: MIT
Signed-off-by: Jakub Sztandera <kubuxu@protonmail.ch>

### CODE REVIEW

File: blocks/blockstore/caching.go [hunk 1]
```diff
@@ -8,14 +8,14 @@ import (
...
 type CacheOpts struct {
-	HasBloomFilterSize   int // 1 bit
+	HasBloomFilterSize   int // 1 byte
 	HasBloomFilterHashes int // No size, 7 is usually best, consult bloom papers
 	HasARCCacheSize      int // 32 bytes
 }
 
 func DefaultCacheOpts() CacheOpts {
 	return CacheOpts{
-		HasBloomFilterSize:   512 * 8 * 1024,
```
Comment (whyrusleeping) [inline blocks/blockstore/caching.go]: I prefer using shift notation for these: `512 << 10`

**Comment by whyrusleeping** [comment]
One small nitpick, doesnt need to be addressed, LGTM

**Comment by Kubuxu** [comment]
Addressed.

**Comment by whyrusleeping** [comment]
Thanks!

**Pull request merged**